CDCgov/prime-reportstream
# Oo/oslynn/6188/co no longer requires negative result

This PR using quality filter to filter out the negative Antigen test for CO.

Test Steps:
1. ./Download test csv input file below.  Note, the input file contains 5 messages and 1 message is a negative Antigen test.
2. ./prime data --input pdi-covid-19-co.csv --input-schema primedatainput/pdi-covid-19 --output-dir junk/output --route
3. Check the HL7 output file and you should see only 4 messages since the negative Antigen test is filtered out.

## Changes
- *Include a comprehensive list of changes in this PR*
- *(For web UI changes) Include screenshots/video of changes*

## Checklist

### Testing

Input Test File:   [pdi-covid-19-co.csv](https://github.com/CDCgov/prime-reportstream/files/10020181/pdi-covid-19-co.csv)

### Process

## Linked Issues
[- Fixes #issue](https://app.zenhub.com/workspaces/onboarding--operations-6166edbd409257001e09f1b2/issues/cdcgov/prime-reportstream/6188)

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue

## Specific Security-related subjects a reviewer should pay specific attention to
- Does this PR introduce new endpoints?
    - new endpoint A
    - new endpoint B
- Does this PR include changes in authentication and/or authorization of existing endpoints?
- Does this change introduce new dependencies that need vetting?
- Does this change require changes to our infrastructure?
- Does logging contain sensitive data?
- Does this PR include or remove any sensitive information itself?

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?
